documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+
+The `FeatureInfo` class is used to provide metadata about each feature, such as its label, key, type, and units if applicable.
+
+```python
+areas = data[AREA_COLUMN].to_numpy()
+locations = data[LOCATION_COLUMN].to_numpy()
+radii = data[RADIUS_COLUMN].to_numpy()
+
+# Additional metadata can be provided for each feature, which will be shown
+# when interacting with it in the viewer.
+area_info = FeatureInfo(
+    label="Area",
+    key="area",
+    type=FeatureType.CONTINUOUS,
+    unit="px²",
+)
+radius_info = FeatureInfo(
+    label="Radius",
+    key="radius",
+    # Discrete features are used for integers.
+    type=FeatureType.DISCRETE,
+    unit="px",
+)
+location_info = FeatureInfo(
+    label="Location",
+    key="location",
+    # Categorical features are used for string-based labels.
+    type=FeatureType.CATEGORICAL,
+    # Categories can be auto-detected from the data, or provided manually
+    # if you want to preserve a specific order for the labels.
+    categories=["top", "middle", "bottom"],
+)
+writer.write_feature(areas, area_info)
+writer.write_feature(radii, radius_info)
+writer.write_feature(locations, location_info)
+```
+
+#### 5. Write the images
+
+The `ColorizerDatasetWriter` writes images as PNGs with encoded object IDs. You can see more about what this looks like and how it works in our [data format documentation](./DATA_FORMAT.md#5-frames). As previously noted, all object IDs must be unique too, so this next section will perform three tasks:
+
+1. Load in the image data from the segmentation images
+2. Remap the object IDs to be unique across all timepoints
+3. Write the remapped images using `ColorizerDatasetWriter.write_image()`
+
+```python
+# Group data by the timestamp
+data_grouped_by_time = data.groupby(TIMES_COLUMN)
+frame_paths = []
+
+for frame_num, frame_data in data_grouped_by_time:
+    # Get the path to the image and load it.
+    frame_path = frame_data.iloc[0][SEGMENTED_IMAGE_COLUMN]
+    segmentation_image = BioImage("raw_dataset/" + frame_path).get_image_data(
+        "YX", S=0, T=0, C=0
+    )
+    # NOTE: For datasets with 3D segmentations, you may need to flatten the data into
+    # 2D images. Typically, it's simplest to do so with a max projection, but may vary
+    # based on your data. Replace the above line with the following:
+    #
+    # segmentation_image = bioio.BioImage(frame_path).get_image_data("ZYX", S=0, T=0, C=0)
+    # segmentation_image = segmentation_image.max(axis=0)
+    #
+    # Remap the segmented so object IDs are unique across all timepoints.
+    (remapped_segmentations, _lut) = remap_segmented_image(
+        segmentation_image, frame_data, OBJECT_ID_COLUMN, INDEX_COLUMN
+    )
+    # Write the new segmentation image.
+    frame_prefix = "frame_"
+    frame_suffix = ".png"
+    writer.write_image(remapped_segmentations, frame_num, frame_prefix, frame_suffix)
+    frame_paths.append(frame_prefix + str(frame_num) + frame_suffix)
```

Comment:
Observation for later: users would not have to do this line if write_image returned this string value `frame_prefix + str(frame_num) + frame_suffix` (since write_image was passed in all those params)

Reply:
Good point, thank you